### PR TITLE
Redundancy information new modules

### PR DESCRIPTION
**Pull request by gianarb**
Now the sidebar contain a `New Modules` box but it returns the same information of the main page..

I have a lot of proposal for the sidebar

* [ ] ~~**Apigility banner**.. Maybe we can load the modules into the site!! @ezimuel~~
* [ ] ~~**ZF Intro training banner** see [ZFsite](http://framework.zend.com)~~
* [x] **Thanks Roave for the server** slogan
* [x] A free banner to promote PHP or ZF event into the world

### CODE REVIEW

**Comment by gianarb** [comment]
@localheinz @Ocramius @ins0 feedback!? :)

**Comment by ins0** [comment]
i would be for ``A free banner to promote PHP or ZF event into the world`` - also for roave but it might gets complicated with zend but idk :smile:

**Comment by gianarb** [comment]
I don't know the policy :) 

ping @weierophinney and @EvanDotPro for this stuff :)

File: module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php [hunk 1]
```diff
@@ -42,17 +42,6 @@ public function testIndexActionCanBeAccessed()
             ->willReturn(0)
         ;
 
-        $moduleMapper
```
Comment (Ocramius) [inline module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php]: Why was this changed?

File: module/Application/view/application/index/index.phtml [hunk 2]
```diff
@@ -46,7 +46,22 @@
 </div>
 <div class="hidden-xs hidden-sm col-md-4">
     <div class="sidebar">
-        <?php echo $this->newModule() ?>
-        <?php echo $this->newUsers() ?>
+        <div class="block">
+            <div class="row">
+                <div class="col-md-12">
+                    <img class="img-responsive" src="http://placehold.it/300x150&text=PHPorZF%20conference">
```
Comment (Ocramius) [inline module/Application/view/application/index/index.phtml]: Escaping is incorrect here: `&amp;` needed

File: module/Application/view/application/index/index.phtml [hunk 3]
```diff
@@ -46,7 +46,22 @@
 </div>
 <div class="hidden-xs hidden-sm col-md-4">
     <div class="sidebar">
-        <?php echo $this->newModule() ?>
```
Comment (Ocramius) [inline module/Application/view/application/index/index.phtml]: Why was this removed?

File: module/Application/view/application/index/index.phtml [hunk 4]
```diff
@@ -46,7 +46,22 @@
...
+        <div class="block">
+            <h4><strong>Hosting by</strong></h4>
+            <hr>
+            <div class="row">
+                <div class="col-md-12">
+                    <a target="_blank" href="https://roave.com"><img class="img-responsive" src="/img/roave-logo-tiny.svg"></a>
+                </div>
+            </div>
+        </div>
+        <?php echo $this->newUsers(); ?>
```
Comment (Ocramius) [inline module/Application/view/application/index/index.phtml]: `<?=` ftw

File: module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php (hunk 1 above)
Comment (gianarb) [inline module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php]: This query was called by $this->newModule() and I have remove it

File: module/Application/view/application/index/index.phtml (hunk 3 above)
Comment (gianarb) [inline module/Application/view/application/index/index.phtml]: Because the list of module is the same of page.. Same informations in the same page

Comment (Ocramius) [inline module/Application/view/application/index/index.phtml]: Gotcha

File: module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php (hunk 1 above)
Comment (Ocramius) [inline module/Application/test/ApplicationTest/Integration/Controller/IndexControllerTest.php]: Fair enuff

**Comment by Ocramius** [comment]
> * [ ] **Apigility banner**.. Maybe we can load the modules into the site!! @ezimuel 

This site has little to do with Apigility: I wouldn't add it here

 > * [ ] **ZF Intro training banner** see [ZFsite](http://framework.zend.com)

Meh, this website is community-maintained, let's not add spam in here.

I'd rather have banners from @localheinz's, @ins0's and your company's in here rather than that stuff.

**Comment by gianarb** [comment]
Np for me, in my opinion the sidebar is a good place to design this site how community of developers.. This is only proposal of message.. But if they are other proposal for me is ok! :+1:

**Comment by Ocramius** [comment]
@gianarb the point is that Zend != community. Community is community, we have enough Zend on this website already (IMO).

People are free to disagree with me, and as a Zend trainer, I'm also working against my own $$$ interests here: I just think that it is correct to highlight who is actually doing the hard work here.

**Comment by gianarb** [comment]
I'm agree with you! :) I wait the proposal by it! :dancer:

**Comment by weierophinney** [comment]
> I just think that it is correct to highlight who is actually doing the hard work here.

Totally. As this is site/project is not hosted by Zend, not maintained by Zend, and not sponsored by Zend, I'd personally feel uncomfortable with Zend banners on the site. It's a community site.

File: module/Application/view/application/index/index.phtml [hunk 5]
```diff
@@ -46,7 +46,22 @@
 </div>
 <div class="hidden-xs hidden-sm col-md-4">
     <div class="sidebar">
-        <?php echo $this->newModule() ?>
-        <?php echo $this->newUsers() ?>
+        <div class="block">
```
Comment (Ocramius) [inline module/Application/view/application/index/index.phtml]: Please remove this for now

Comment (gianarb) [inline module/Application/view/application/index/index.phtml]: Done b6c66b7

**Comment by Ocramius** [comment]
Merging for now. @localheinz @ins0 and others who contributed to this website in large amounts: please feel free to add your own company logos.

**Comment by gianarb** [comment]
maybe we can design a partners/contributors page!!

**Comment by Ocramius** [comment]
@gianarb I'd rather rotate the banners based on contributions % :P

**Comment by gianarb** [comment]
:+1: Thanks! :crescent_moon:
